tests/load-tests/collect-pod-logs/README.md

```diff
@@ -0,0 +1,14 @@
+# JIRA task - https://issues.redhat.com/browse/STONE-831 
+# The bash script is collect-openshift-pod-logs.sh
+
+
+The script logic:
+================
+Log directory configuration
+Format:  ./snapshot-collect-pod-logs.sh logs-${USER}-$(date +%Y-%m-%d)  
+if no parameter given, a default will be used for the log_dir - log_dir="logs-${USER}-$(date +%Y-%m-%d-%H-%M-%S)"
+
+1. The script will run in  a contiuous loop iterating through all namespaces/pods/container
+2. All log files will be stored under a main log_dir of type log_dir
+3. Per each container log, if not empty - existing container log will be removed and the new one to be stored in the 
+   proper hierarchy
```

Comment:
```suggestion
## JIRA Task
[STONE-831](https://issues.redhat.com/browse/STONE-831)

## Bash Script Name
The bash script is `collect-openshift-pod-logs.sh`


## Log directory configuration
Format: `./snapshot-collect-pod-logs.sh logs-${USER}-$(date +%Y-%m-%d)`
If no parameter is given, a default log_dir will be used: `log_dir="logs-${USER}-$(date +%Y-%m-%d-%H-%M-%S)"`
## Script Logic
1. The script will run in a continuous loop iterating through all namespaces/pods/container.
2. All log files will be stored under a main log_dir of type `log_dir`.
3. For each container log, if not empty:
   - Existing container log will be removed.
   - The new log will be stored in the proper hierarchy.
```

Here's how it looks (sorry, don't know a better way to suggest changes in Markdown):
## JIRA Task
[STONE-831](https://issues.redhat.com/browse/STONE-831)

## Bash Script Name
The bash script is `collect-openshift-pod-logs.sh`


## Log directory configuration
Format: `./snapshot-collect-pod-logs.sh logs-${USER}-$(date +%Y-%m-%d)`
If no parameter is given, a default log_dir will be used: `log_dir="logs-${USER}-$(date +%Y-%m-%d-%H-%M-%S)"`
## Script Logic
1. The script will run in a continuous loop iterating through all namespaces/pods/container.
2. All log files will be stored under a main log_dir of type `log_dir`.
3. For each container log, if not empty:
   - Existing container log will be removed.
   - The new log will be stored in the proper hierarchy.

Reply:
Thanks, updated most of your suggestions